earthpy/tests/test_hillshade.py

```diff
@@ -0,0 +1,61 @@
+""" Tests for the hillshade function. """
+
+import numpy as np
+import pytest
+import earthpy.spatial as es
+
+
+@pytest.fixture
+def hillshade_arr():
+    """Create three-dimensional array for testing input array
+    to hillshade function. """
+    arr = np.array(
```

Comment:
Ok a few suggestions on this array
1. you squeeze it below to a 2dim array twice. Why not create a 2-dim arr here and add an axis for the first test. 
2. For the docstring how about `2-dimensional array of shape (6,6)`

Reply:
Great point. This has now been implemented.